ephypype/nodes/utils.py

```diff
@@ -73,3 +73,21 @@ def create_datagrabber(data_path, template_path, template_args):
     datasource.inputs.sort_filelist = True
 
     return datasource
+
+
+def get_frequency_band(freq_band_names, freq_bands):
+
+    from ephypype.aux_tools import get_freq_band
+
+    get_frequency_node = pe.Node(interface=Function(
+            input_names=['freq_band_name',
+                         'freq_band_names',
+                         'freq_bands'],
+            output_names=['freq_bands'],
+            function=get_freq_band),
+            name='get_frequency_node')
+
+    get_frequency_node.inputs.freq_band_names = freq_band_names
+    get_frequency_node.inputs.freq_bands = freq_bands
+
+    return get_frequency_node
```

Comment:
I added this function to replicate some lines of the code we used in 
https://neuropycon.github.io/neuropycon_doc/examples/conn_graph_example.html#conn-graph-example

In particular this code
`def get_freq_band(freq_band_name)`:

    from params_congraph import freq_band_names, freq_bands

    if freq_band_name in freq_band_names:
        print freq_band_name
        print freq_band_names.index(freq_band_name)

        return freq_bands[freq_band_names.index(freq_band_name)]`



` main_workflow.connect(infosource, ('freq_band_name', get_freq_band),
                          spectral_workflow, 'inputnode.freq_band')`

is replaced by 
https://github.com/neuropycon/ephypype/blob/5a4594773154f2410852ce248eb8589141a1cdcb/examples/plot_connectivity.py#L85

https://github.com/neuropycon/ephypype/blob/5a4594773154f2410852ce248eb8589141a1cdcb/examples/plot_connectivity.py#L98-L99

https://github.com/neuropycon/ephypype/blob/5a4594773154f2410852ce248eb8589141a1cdcb/examples/plot_connectivity.py#L115-L116

...but, I don't like this solution...

Reply:
@annapasca why don't you like this solution?

Reply:
> @annapasca why don't you like this solution?

too tricky, but if you think it's ok, I'm happy :)

Reply:
you mean tricky to explain why you don't like it? Or that it's too tricky for the users to understand? We can go with this for now and simplify more later ...

Reply:
> you mean tricky to explain why you don't like it? Or that it's too tricky for the users to understand? We can go with this for now and simplify more later ...

from the user view point...but you are right...we'll simplify later :)